app/services/slack_service.rb

```diff
@@ -151,9 +151,9 @@ def self.get_receivers_from_command(text)
   end
 
   def self.parse_amount(text)
-    amount = text.split(' ').last
+    amount = text.split(">").last.split("for").first
```

Comment:
that somehow looks wired and fragile - just a feeling

Reply:
Isn't message parsing always a bit fragile though 😄? 

Kidding aside, I'm trying to select the text between the last '>' char and the the 'for' keyword because a command would look like this:
`/kudo @someone @someone_else 10 for helping me with reviews`

I looked at a regex expression which would do the same as the code above but I preferred this because it is very readable in my opinion.